.github/workflows/test.yml

```diff
@@ -0,0 +1,74 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test OpenUpgrade migration
+
+on:
+  push:
+    branches: ["14.0*"]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DB: "openupgrade"
+      DB_USERNAME: "odoo"
+      DB_PASSWORD: "odoo"
+      DB_HOST: "localhost"
+      DB_PORT: 5432
+      ODOO_REPO: "odoo/odoo"
+      ODOO: "./odoo-bin"
```

Comment:
I think the current error "/home/runner/work/_temp/9122709a-260c-40af-971b-85055f6597eb.sh: line 6: ./odoo-bin: No such file or directory" is because after the checkout you are not inside the odoo repo, but just outside of it. A simple way to fix this woulde be to set `ODOO: "odoo/odoo-bin"` instead or alternatively tweak the run command line 74.

Reply:
tried, but no

Reply:
my bad sorry. So as you can see from the logs,  2 lines before the sed commands work. It means that at this point of the script we are in this directory: https://github.com/OCA/OpenUpgrade/tree/13.0/odoo/openupgrade and the odoo-bin scripts is 2 levels before:
So what about setting instead
`ODOO: "../../odoo-bin"` ?

Reply:
tried, but no

Reply:
It's quite mysterious why it should be any different from https://github.com/OCA/OpenUpgrade/blob/13.0/.github/workflows/test.yml apart from the versions.

Reply:
feel free to push here is you want to try things :S

Reply:
May be before line 74 here `.github/workflows/test.yml` we could add these lines:
```
pwd
ls -la
```
to figure out better were we are and what is available to us at this point...

Reply:
Gotta cook, but will have a look later

Reply:
OMG did we all forget that OpenUpgrade 14 is no longer a fork of Odoo.

Reply:
This works, for this job: https://github.com/OCA/OpenUpgrade/pull/2747/commits/be990059ba6a7d1f0883afe933262f5d5a3f837e. Do you want to continue the effort or shall I?

Reply:
Cannot you push into this branch? We just need to fix the documentation.

Reply:
I cannot push to this branch it seems.